src/modules/Channel/components/MessageList/index.tsx

```diff
@@ -173,7 +176,22 @@ export const MessageList = ({
   if (allMessagesFiltered.length < 1) {
     return renderPlaceholderEmpty();
   }
+  const handleUnreadCountClick = () => {
+    if (scrollRef?.current) scrollRef.current.scrollTop = scrollRef.current.scrollHeight;
+    if (!disableMarkAsRead && !!currentGroupChannel) {
+      markAsReadScheduler.push(currentGroupChannel);
+      messagesDispatcher({
+        type: messageActionTypes.MARK_AS_READ,
+        payload: { channel: currentGroupChannel },
+      });
+    }
+    setInitialTimeStamp(null);
+    setAnimatedMessageId(null);
+    setHighLightedMessageId(null);
+  };
 
+  const isUnreadNotificationButtonDisplayed = !!((!isScrollBottom || hasMoreNext) && (unreadSince || unreadSinceDate));
+  const isScrollToBottomButtonDisplayed = scrollBottom > SCROLL_BOTTOM_PADDING;
```

Comment:
nit: I feel like this is easier to parse:

` const isUnreadNotificationButtonDisplayed = (!isScrollBottom || hasMoreNext) && (!!unreadSince || !!unreadSinceDate);`